Userland/Libraries/LibGL/SoftwareGLContext.cpp

```diff
@@ -3017,4 +3061,151 @@ void SoftwareGLContext::sync_device_texcoord_config()
     m_rasterizer.set_options(options);
 }
 
+void SoftwareGLContext::gl_lightf(GLenum light, GLenum pname, GLfloat param)
+{
+    APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED(gl_lightf, light, pname, param);
+
+    RETURN_WITH_ERROR_IF(m_in_draw_state, GL_INVALID_OPERATION);
+    RETURN_WITH_ERROR_IF(light < GL_LIGHT0 || light > GL_LIGHT7, GL_INVALID_ENUM);
```

Comment:
Sure, I figured since we hard-coded these for the Software Context it wouldn't be an issue because we support the minimum number of lights (8), however it makes more sense to perform that check instead.